galasa-ui/src/utils/constants/common.ts

```diff
@@ -55,9 +55,26 @@ const RESULTS_TABLE_COLUMNS = [
   { id: "result", columnName: "Result" },
 ];
   
+const PARAMS = {
```

Comment:
Very minor - instead of `PARAMS`, this could be more descriptive and be something like `RUN_QUERY_PARAMS`?

Feel free to come up with something better!